tests/test-gpio.js

```diff
@@ -21,7 +26,7 @@ var mark = 0;
 
 var edgeInterval = setInterval(function () {
     var count = 0;
-    pinA = gpio.open({ pin: pins.IO7 });
+    pinA = gpio.open({pin: pins.IO7});
```

Comment:
i actually prefer the leaving the spaces before, is there a reason to change this?